<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Bikash" failures="1" tests="6" name="Homepage Test" time="22.43" errors="0" timestamp="13 Feb 2020 18:16:48 GMT">
  <testcase classname="com_TestCase.HomePage_TestCase" name="companiesButtonPresentValidation" time="12.04"/>
  <testcase classname="com_TestCase.HomePage_TestCase" name="buttonsPresentValidation" time="0.279"/>
  <testcase classname="com_TestCase.HomePage_TestCase" name="CombinedButtonClickValidation" time="0.082">
    <skipped/>
  </testcase> <!-- CombinedButtonClickValidation -->
  <testcase classname="com_TestCase.HomePage_TestCase" name="CombinedButtonClickValidation" time="0.067">
    <skipped/>
  </testcase> <!-- CombinedButtonClickValidation -->
  <testcase classname="com_TestCase.HomePage_TestCase" name="CombinedButtonClickValidation" time="0.064">
    <skipped/>
  </testcase> <!-- CombinedButtonClickValidation -->
  <testcase classname="com_TestCase.HomePage_TestCase" name="CombinedButtonClickValidation" time="0.061">
    <failure type="java.lang.AssertionError" message="expected [CRMPRO0] but found [CRMPRO]">
      <![CDATA[java.lang.AssertionError: expected [CRMPRO0] but found [CRMPRO]
at com_TestCase.HomePage_TestCase.CombinedButtonClickValidation(HomePage_TestCase.java:46)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:835)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- CombinedButtonClickValidation -->
</testsuite> <!-- Homepage Test -->
